src/fprime/cookiecutter_templates/cookiecutter-fprime-component/hooks/post_gen_project.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import print_function
+
+import datetime
+import os
+from os.path import join
+from fprime.fbuild.settings import IniSettings
+from fprime.fbuild.builder import Build
+from pathlib import Path
+import textwrap
+
+
+def replace_contents(filename, what, replacement, count=1):
+    with open(filename) as fh:
+        changelog = fh.read()
+    with open(filename, "w") as fh:
+        fh.write(changelog.replace(what, replacement, count))
+
+
+def remove_line(filename, removal):
+    with open(filename, "r") as f:
+        lines = f.readlines()
+        f.close()
+    with open(filename, "w") as f:
+        for line in lines:
+            if line != removal:
+                f.write(line)
+        f.close()
+
+
+def update_sdd():
+    ports = "## Port Descriptions\n| Name | Description |\n"
+    if (
+        "{{cookiecutter.component_kind}}"  # lgtm [py/comparison-of-constants]  lgtm [py/constant-conditional-expression]
```

Comment:
I just thought of an idea that might allow you to remove all these manual lgtm warning suppression comments. What you make all cookiecutter inputs to this function parameters, then call the function with the cookiecutter options.

Ex:

```
def update_sdd(component_kind):
    ...

update_sdd("{{cookiecutter.component_kind}}")
```